tools/README.md

```diff
@@ -0,0 +1,2 @@
+To collect logs manually, please use the collect_archive_logs.sh and/or collect_current_logs.sh scripts depending on the information that is requested.  
+Use the exclude_files file to add any files you don't want included in log collection.
```

Comment:
Since we have a `README.md` that is specific to this script, I think we should move this and all files into it's own `subdirectory`, or at the very least rename the `README.md` to something like `README.collect_logs.md`.